### ISSUE TITLE
Fold entity only shows collapse / expand arrows (iOS12)

### ISSUE THREAD

**Issue by iemand-dev** [thomasloven/lovelace-fold-entity-row#188]
Hi

I'm experiencing issues with the fold-entity-row specifically on iOS12 where the UI only shows the expand / collapse arrow buttons. The labels, toggles and icons are "gone". Using the expand or collapse arrows does nothing other than rotate the arrows.
Doesn't matter if I run Home Assistant in the browser or in the app, result is the same.
Issue occurs on my iPad Air with iOS 12. On an iPad 8th gen running iOS 15 (or desktop with Firefox), the issue does not present itself.
Downgrading to version 20.0.9 and the issue is gone.

Tested on:
Device: iPad Air (running iOS 12.5.5)
Home assistant version: 2021.10.6
fold-entity-row: 20.0.11 (started occurring with 20.0.10)

```
type: entities
entities:
  - type: custom:fold-entity-row
    head: light.kitchen_lights
    entities:
      - type: custom:slider-entity-row
        entity: light.kitchen_cooking_area
        name: Cooking area
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.kitchen_fridge
        name: Fridge
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.kitchen_hall
        name: Hall
        hide_when_off: true
        toggle: true
  - type: custom:fold-entity-row
    head: light.dining_room_lights
    entities:
      - type: custom:slider-entity-row
        entity: light.living_room_main
        name: Main
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.living_room_table
        name: Dining table
        hide_when_off: true
        toggle: true
  - type: custom:fold-entity-row
    head: light.living_room_lights
    entities:
      - type: custom:slider-entity-row
        entity: light.tv_room_main
        name: Main
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.tv_room_light_stand
        name: Stand
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.tv_room_wall_left
        name: Wall left
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.tv_room_wall_right
        name: Wall right
        hide_when_off: true
        toggle: true
      - type: custom:slider-entity-row
        entity: light.tv_light_stand
        name: TV light stand
        hide_when_off: true
        toggle: true
  - type: custom:fold-entity-row
    head: light.bedroom_lights
    entities:
      - type: custom:slider-entity-row
        entity: light.bedroom_bedside_table
        name: Stand
        hide_when_off: true
        toggle: true
```

On iOS 12
![fold entity ios](https://user-images.githubusercontent.com/63008780/141659571-081920ed-a8ac-4d09-bf21-3e26f222b0fb.png)

On Windows 10 (Firefox)
![fold entity firefox](https://user-images.githubusercontent.com/63008780/141659597-e1add5d9-745e-464a-88e8-45b828b7f6cc.png)

**Comment by thomasloven** [comment]
Please try with the latest version of Home Assistant and clear your browser caches.

**Comment by iemand-dev** [comment]
Upgraded HA to 2021.11.5 and Fold entity row to 2.0.12.
Issue still persists and reloaded without use of browser cache.

Connected device on Mac to take a look in dev tools and found the following. Don’t know if this helps or if this is the source of the issue.
<img width="711" alt="52E8319C-090D-4961-B7AA-97244D4A8601" src="https://user-images.githubusercontent.com/63008780/142768532-30f2c1d0-48dd-4596-a089-087f6d7c8dd6.png">

**Comment by iemand-dev** [comment]
Issue no longer present.

**Comment by VossenTech** [comment]
@iemand-dev did you do something specific to fix it? I am running HA on 2021.11.5 and Fold entity row on 2.0.12 but still seeing the issue.

**Comment by iemand-dev** [comment]
> @iemand-dev did you do something specific to fix it? I am running HA on 2021.11.5 and Fold entity row on 2.0.12 but still seeing the issue.

I was surprised that it suddenly worked as expected.
I was running HA 2021.10.6 and fold entity 20.0.11 when reporting the issue.
I reverted back to fold entity 20.0.9 and the issue was gone.
After the comment of the developer I upgraded HA to 2021.11.5 and fold entity 20.0.12 and it still occurred.
Magically a few days later the issue was gone. Can’t say I did anything special.

I’m using Safari to control HA.
I’m having different issue when using the HA app (app goes black).
